Model/Image.php

```diff
@@ -22,19 +22,9 @@ public function __construct(
         $this->storeManager = $storeManager;
     }
 
-    public function getDefaultUrl(string $currentUrl): string
+    public function getCustomUrl(string $imageUrl, int $width, int $height): string
     {
-        return $this->getServiceUrl($currentUrl, $this->config->getConfigValue(Config::XPATH_FIELD_LARGE));
-    }
-
-    public function getSmallUrl(string $currentUrl): string
-    {
-        return $this->getServiceUrl($currentUrl, $this->config->getConfigValue(Config::XPATH_FIELD_SMALL));
-    }
-
-    public function getAutoCompleteUrl(string $currentUrl): string
-    {
-        return $this->getServiceUrl($currentUrl, $this->config->getConfigValue(Config::XPATH_FIELD_AUTOCOMPLETE));
+        return $this->getServiceUrl($imageUrl, 'w=' . $dimensions['width'] . '&h=' . $dimensions['height'] . '&auto=compress&auto=format');
```

Comment:
you don't need `$dimensions[]` here, go with `$width` and `$height`

Reply:
The possibility to have some extra options `&auto=compress&auto=format` after for instance, maybe something to reconsider

Reply:
Made a whoopsie there! Will add other options like cropping with selector in the admin later